<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>518</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Bold">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Italic">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Underline">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FGColor">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>89</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: right;
padding-right: 15%;</string>
   </property>
   <property name="text">
    <string>前景色</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BGColor">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>89</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: right;
padding-right: 15%;</string>
   </property>
   <property name="text">
    <string>背景色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_FontSize">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>字号(&amp;S)：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_FontFamily">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>字体(&amp;F)：</string>
   </property>
   <property name="buddy">
    <cstring>fontComboBox</cstring>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>226</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
